packages/components/src/components/code-snippet/_code-snippet.scss

```diff
@@ -50,6 +52,37 @@
       outline: none;
       border: 2px solid $focus;
     }
+
+    &::before {
+      @include tooltip--caret;
+      display: none;
+    }
+
+    .#{$prefix}--copy-btn__feedback {
+      @include tooltip--content('icon');
+      clip: auto;
+      margin: auto;
+      overflow: visible;
+      display: none;
+    }
+    @include tooltip--placement('icon', 'bottom', 'center');
+
+    &.#{$prefix}--copy-btn--animating::before,
```

Comment:
I know this was merged but is there any way we can un-nest styles?

Reply:
will be addressed in https://github.com/carbon-design-system/carbon/pull/5006